docs/debugging-restores.md

```diff
@@ -99,5 +100,15 @@ Both errors and warnings are structured in the same way:
 
 * `Namespaces`: A map of namespaces to the list of issues related to the restore of their respective resources.
 
+## Restoring a load balancer Service
```

Comment:
I would prefer to see this in a "Known Issues" section/page - wdyt?

Reply:
yes. that's my preference too, but I'm also used to putting known issues into a changelog/release notes. Is this sufficient?

Reply:
This type of information isn't really appropriate for a changelog or release notes. It doesn't affect a single version.

Reply:
got it. separate section/page it is